frontend-v2/src/lib/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { clsx, type ClassValue } from "clsx";
+import { twMerge } from "tailwind-merge";
+
+export function cn(...inputs: ClassValue[]) {
```

Comment:
what does CN stand for? would be nice to have a comment that introduces this function without having to infer from usage / dig into the dependencies' docs, however briefly. no need to explain everything completely since it's obviously just wrapping something else.

Reply:
this util is generated by shadn. i'll add a link to an explanation of it.